src/api/Feed.js

```diff
@@ -162,6 +162,56 @@ class Feed extends Base {
         this.updateFeedItem({ ...annotation, isPending: false }, id);
     }
 
+    updateAnnotation = (
+        file: BoxItem,
+        annotationId: string,
+        text: string,
+        permissions: AnnotationPermission,
+        successCallback: (annotation: Annotation) => void,
+        errorCallback: ErrorCallback,
+    ): void => {
+        const annotationData = {
+            message: text,
+        };
+
+        this.annotationsAPI = new AnnotationsAPI(this.options);
+
+        if (!file.id) {
```

Comment:
Should this happen on the first line of this method, before the annotations API instance is created?

Reply:
Yes that makes sense.